js/src/sheet.ts

```diff
@@ -97,14 +94,6 @@ let SheetModel = widgets.DOMWidgetModel.extend({
         this.set('data', data);
         this.save_changes()
     },
-    cell_to_grid: function(cell, save) {
```

Comment:
Just trying to remove noise in the code so that it's easier to code, @maartenbreddels was it an important method? Or just a leftover?